src/data/eventregistration.json

```diff
@@ -3,6 +3,8 @@
         "headline": {
             "title": "Create QR code for event check-in"
         },
+        "intro": "The CWA now allows check-in for events and venues. People who host events or have a business can now create a QR code. By scanning the QR code, guests can check in upon arrival to register si their attendance. If desired, the app will also create a corresponding diary entry. If a checked-in person later tests positive for the coronavirus, other people who were checked in at the same time can be warned.",
```

Comment:
"[…] to register si their […]" -> "[…] to register their […]"

Reply:
Done!

Reply:
The Corona-Warn-App